gen_server.sh

```diff
@@ -1,10 +1,7 @@
 rm -rf server/schema
```

Comment:
Except for the yarn installs, everything in the Makefile scripts happens in `server`.  Can the root dir be cleaned up and everything including the Makefile be put in the `server` directory?

Reply:
The issue is we use files from the `//schema/` directory in the generation scripts. I would prefer to move the scripts out of root though.

Reply:
Oh i see... this line: 
`cp -r schema/ server/schema/`

could be

`cp -r ../schema/ .`